.github/workflows/main-doc.yaml

```diff
@@ -0,0 +1,98 @@
+name: Main CI WorkFlow
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+    paths:
+      - '**.md'
+      - 'docs/**'
+      - '**/OWNERS'
+      - '**/MAINTAINERS'
+  pull_request:
+    paths:
+      - '**.md'
+      - 'docs/**'
+      - '**/OWNERS'
+      - '**/MAINTAINERS'
+
+jobs:
+  lint:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    name: Verify vendor, licenses, do lint
+    env:
+      GOPATH: /home/runner/work/${{ github.repository }}
+    steps:
+      - run: 'echo "only docs modified, no need to trigger CI"' 
```

Comment:
Do we need to run lint here? Now we have run the whitenoise in lint IIRC, or later run other lint check for doc :) @gy95

Reply:
good catch